src/main/clojure/clara/rules/compiler.clj

```diff
@@ -3,17 +3,11 @@
    This is the Clara rules compiler, translating raw data structures into compiled versions and functions.
    Most users should use only the clara.rules namespace."
   (:require [clara.rules.engine :as eng]
-            [clara.rules.listener :as listener]
```

Comment:
Were these just unused ns's to clean up?

Reply:
yes, might as well clean up while im here